src/entities/tag.cpp

```diff
@@ -469,7 +482,8 @@ bool Tag::store() {
     if (this->id > 0) {
         query.prepare(
                 "UPDATE tag SET name = :name, priority = :priority, "
-                        "parent_id = :parentId, " + colorField + " = :color "
+                        "parent_id = :parentId, " + colorField + " = :color, "
+                        "created = datetime('now') "
```

Comment:
No, not in your code, I mean more general, philosophically speaking: "If something is change the old is destroyed and something new emerges" :)

I wasn't aware of that problem in sqlite. However, it's possible to create a new table with the desired structure, convert the values and drop the old table. The question stands: do you prefer saving (imho mostly duplicate) data over that?